adapters/adapters-base/build.gradle

```diff
@@ -52,6 +52,7 @@ dependencies {
     compileOnly('org.jetbrains:annotations:23.0.0')
     compileOnly('com.datastax.cassandra:cassandra-driver-core:3.11.3')
     implementation("org.slf4j:slf4j-api:${project.slf4jVersion}")
+    implementation("com.google.inject:guice:${guiceVersion}")
```

Comment:
only to make injection annotation available for `ConnectedClientsSchema`.

I am open to relocate the class too.